Revit_Core_Engine/Create/View/ViewReflectedCeilingPlan.cs

```diff
@@ -92,7 +92,7 @@ public static View ViewReflectedCeilingPlan(this Document document, Autodesk.Rev
                     BH.Engine.Base.Compute.RecordWarning("There is already a view named '" + viewName + "'." + " It has been named '" + result.Name + "' instead.");
```

Comment:
This can be avoided by using the same approach suggested for sheet number and plan view name. The same check should take place here before setting the new view name.